ChattoAdditions/sources/Common/ScreenMetric.swift

```diff
@@ -73,6 +79,10 @@ private extension ScreenMetrics {
             return 812
         case .inch6_1, .inch6_5:
             return 896
+        case .inch6_0_6:
+            return 844
```

Comment:
Do you think we could add here the reference of the source where we grabbed this values from?